conformance/tdl/if_multi.ion

```diff
@@ -0,0 +1,132 @@
+         (each "an expression group with multiple values"
+               (mactab (macro is_multi () (.$ion::if_multi (.. 0 1 2) true false)))
+               "a macro that produces multiple values"
+               (mactab (macro is_multi () (.$ion::if_multi (.values (.. 0 1 2)) true false)))
+               "an expression group with a single macro that produces multiple values"
+               (mactab (macro is_multi () (.$ion::if_multi (..(.values (.. 0 1 2))) true false)))
+               "an expression group with multiple macros "
+               (mactab (macro is_multi () (.$ion::if_multi (..(.values 0) (.values 1)) true false)))
+               "an expression group with multiple values and some macros that produce nothing"
+               (mactab (macro is_multi () (.$ion::if_multi (.. 0 1 2 (.none)) true false)))
+               "an expression group with multiple values that starts with a macro that produces nothing"
+               (mactab (macro is_multi () (.$ion::if_multi (.. (.none) 0 1 2) true false)))
+               (then "then if_multi should produce the true branch"
+                     (toplevel ('#$:is_multi'))
+                     (produces true))))
+
+(ion_1_1 "when the first argument is multiple values"
+         (then "the 'true_branch' argument can be"
+               (each "an empty expression group"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (..) false)))
+                     "a macro that produces no values"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (.none) false)))
+                     "an expression group with macros that produces no values"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (.. (.none) (.none)) false)))
+                     (then (toplevel 1 ('#$:is_multi') 2)
+                           (produces 1 2)))
+               (each "one value"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") true false)))
+                     "an expression group with one value"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (.. true) false)))
+                     "a macro that produces one value"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (.$ion::values true) false)))
+                     (then (toplevel 1 ('#$:is_multi') 2)
+                           (produces 1 true 2)))
+               (each "an expression group with multiple values"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (.. true true) false)))
+                     "a macro that produces multiple values"
+                     (mactab (macro is_multi () (.$ion::if_multi (.. "two" "things") (.$ion::values (.. true true)) false)))
+                     (then (toplevel 1 ('#$:is_multi') 2)
+                           (produces 1 true true 2))))
```

Comment:
Not necessarily, but I think it's easier for a human to reason about it if you say that there must be a concrete value after the thing we're testing rather than expecting "end of container" after the thing we're testing.